### PR TITLE
AMC-BLDC - test FW

### PR DESCRIPTION
**Pull request by davidetome**
This PR adds test `FW` (on-board tests) which will be used in `IPTS` for board testing

cc @Nicogene @pattacini @marcoaccame

### CODE REVIEW

File: emBODY/eBcode/arch-arm/board/amcbldc/bsp/embot_hw_bsp_amcbldc.h [hunk 1]
```diff
@@ -20,6 +20,7 @@ namespace embot { namespace hw { namespace bsp { namespace amcbldc {
     embot::hw::BTN EXTFAULTbutton();
     
     float getVIN();
+    float getCIN();
```
Comment (marcoaccame) [inline emBODY/eBcode/arch-arm/board/amcbldc/bsp/embot_hw_bsp_amcbldc.h:23]: maybe add also `float getTEMP()` and specify measurement units, maybe celsius degrees?

File: emBODY/eBcode/arch-arm/board/amcbldc/test/src/motorhal2/analog.c [hunk 2]
```diff
@@ -367,6 +367,15 @@ uint32_t analogVph3(void)
 	return VPH_GAIN * vref_mV * adc1_Buffer.vph3 >> 16u;
 }
 
+/*******************************************************************************************************************//**
+ * @brief   Measure temp
+ * @param   void
+ * @return  temp voltage in mV [0 .. 57000mV]
```
Comment (marcoaccame) [inline emBODY/eBcode/arch-arm/board/amcbldc/test/src/motorhal2/analog.c:373]: pls fix measurement units

**Review by marcoaccame — CHANGES_REQUESTED** [review]
Hi @davidetome, the PR is fine. Only thing, pls fix comments on measuremnt units of temperature

**Comment by davidetome** [comment]
ciao @marcoaccame thaks! I did the mods...

File: emBODY/eBcode/arch-arm/board/amcbldc/bsp/embot_hw_bsp_amcbldc.h (hunk 1 above)
Comment (davidetome) [reply emBODY/eBcode/arch-arm/board/amcbldc/bsp/embot_hw_bsp_amcbldc.h:23]: done!

File: emBODY/eBcode/arch-arm/board/amcbldc/test/src/motorhal2/analog.c (hunk 2 above)
Comment (davidetome) [reply emBODY/eBcode/arch-arm/board/amcbldc/test/src/motorhal2/analog.c]: done!

**Review by marcoaccame — APPROVED** [review]
OK, thanks

**Pull request merged**